exercises/wordy/.meta/generator/wordy_case.rb

```diff
@@ -16,7 +16,7 @@ def indent(size, lines)
   end
 
   def assertion
-    return error_assertion unless expected
+    return error_assertion if expected == false
```

Comment:
Where is `false?` defined?

Neither of these will work.

`expected` comes from the canonical data and is either not present or an integer or false.